Jenkinsfile

```diff
@@ -62,7 +62,10 @@ pipeline {
 
     stage('Publish client Docker image') {
       when {
-        branch 'master'
+        allOf {
+          branch 'master'
+          tag "v*"
```

Comment:
@benoaviram when will this happen? what does it mean?

Reply:
Just taking this condition from the publish script to the jenkins file. 
In the past we had this condition in the jenkins file which ran the publish script only on master branch and inside the publish script there was a check for a fixed-version (using the old way - git describe).
From now - the condition resides on the Jenkins file, only on master master branch which has tag (fixed-version) we will run the publish script